pyfstat/helper_functions.py

```diff
@@ -90,24 +89,23 @@ def set_up_logger(outdir=None, label="pyfstat", log_level="INFO"):
         logger.addHandler(stream_handler)
 
     if any([type(h) == logging.FileHandler for h in logger.handlers]) is False:
-        if label:
-            if outdir:
-                if not os.path.exists(outdir):
-                    os.makedirs(outdir, exist_ok=True)
-            else:
-                outdir = "."
```

Comment:
I've also dropped this `else` case since I'd say we shouldn't spam files into any root directory without the user explicitly asking for it.

Reply:
I agree :+1: